### DISCUSSION TITLE
Recommendations for connection pooling with the v1 client?

### DISCUSSION THREAD

**Discussion by johnbellessa** [hyperium/hyper#3683] [Q&A] [answered]
As the title says, I'm looking for recommendations on pooling connections with the v1 client. I'm particularly interested in using the v1 client, as opposed to the legacy client, because I need to perform special handling with the TcpStream before handing it over to the client, which seems a lot more straight forward with v1.

Any advice would be greatly appreciated!

**Comment by seanmonstar** [comment, marked as answer]
The eventual "new" `Client`, but it doesn't exist yet :).

**Reply by johnbellessa** [reply]
Thanks for the reply. Is there an ETA or any design available to review?
